zips/zip-0312.rst

```diff
@@ -3,11 +3,11 @@
   ZIP: 312
   Title: FROST for Spend Authorization Multisignatures
   Owners: Conrado Gouvea <conrado@zfnd.org>
-          Chelsea Komlo <ckomlo@uwaterloo.ca>
-          Deirdre Connolly <deirdre@zfnd.org>
+  Original-Authors: Chelsea Komlo
+                    Deirdre Connolly
   Status: Draft
   Category: Wallet
-  Created: 2022-08-dd
+  Created: 2024-08-dd
```

Comment:
This can just be the date it was originally committed which was apparently 2023-01-12. https://github.com/zcash/zips/commit/48f8c2c80e890385a9619e4dbf3b62ff8963b010
```suggestion
Created: 2023-01-12
```